repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,2836 @@
+        /information_management_staff:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: information_management_staff
+          locale.en.name: Information management staff
+        /library,_knowledge_and_information_services:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: library,_knowledge_and_information_services
+          locale.en.name: Library, knowledge and information services
+        /project_and_programme_management:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: project_and_programme_management
+          locale.en.name: Project and programme management
+      /healthcare_scientists:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: healthcare_scientists
+        locale.en.name: Healthcare scientists
+        /analytical_toxicology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: analytical_toxicology
+          locale.en.name: Analytical toxicology
+        /anatomical_pathology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: anatomical_pathology
+          locale.en.name: Anatomical pathology
+        /audiology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: audiology
+          locale.en.name: Audiology
+        /biomedical_science:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: biomedical_science
+          locale.en.name: Biomedical science
+        /blood_sciences:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: blood_sciences
+          locale.en.name: Blood sciences
+        /cancer_genomics:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: cancer_genomics
+          locale.en.name: Cancer genomics
+        /cardiac_sciences:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: cardiac_sciences
+          locale.en.name: Cardiac sciences
+        /cellular_sciences:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: cellular_sciences
+          locale.en.name: Cellular sciences
+        /clinical_biochemistry:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: clinical_biochemistry
+          locale.en.name: Clinical biochemistry
+        /clinical_bioinformatics__genomics_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: clinical_bioinformatics__genomics_
+          locale.en.name: Clinical bioinformatics (genomics)
+        /clinical_bioinformatics__health_informatics_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: clinical_bioinformatics__health_informatics_
+          locale.en.name: Clinical bioinformatics (health informatics)
+        /clinical_bioinformatics__physical_sciences_:
```

Comment:
Can we remove the trailing `_` from the name and key ?

Reply:
Going through and changing all of these now